postpic/helper.py

```diff
@@ -738,7 +738,7 @@ def linear_phase(field, dx):
     _kspace_propagate_generator.
     '''
     transform_state = field._transform_state(dx.keys())
-    axes = [ax.grid for ax in field.axes]  # each axis object returns new numpy array
```

Comment:
this comment has not been true for a while. So this actually fixes a bug that no one has seen yet.